site/src/main/resources/templates/uploadValues.html

```diff
@@ -0,0 +1,183 @@
+                var values;
+                date1 = document.getElementById("date1").value;
+                date2 = document.getElementById("date2").value;
+                if(date1 == "" || date2== "" )
+                {
+                    alert("wrong input");
+                    return;
+                }
+                var flag = false;
+                if(document.getElementById("prediction").value == 2){
+                    flag = true;
+                }
+                var str = '{ "fromDate": "' + date1 + '", "toDate": "' + date2 + '", "withPrediction": "' + flag.toString() + '" }';
+                var jsonDates = JSON.parse(str);
+                var xhr = new XMLHttpRequest();
+                xhr.open("POST", '/uploadValues', true);
+                xhr.setRequestHeader('Content-type', 'application/json');
+                xhr.onreadystatechange = function() {
+                    if (xhr.readyState == 4 && xhr.status == 200) {
+                        values = xhr.responseText;
+                        button.innerHTML = 'Готово!';
+                        drawChart(values);
+                    }
+                };
+                xhr.send(JSON.stringify(jsonDates));
+                button.innerHTML = 'Loading...'; // (2)
+                button.disabled = true;
+            }
+
+            function drawChart(values) {
+                if(values == null){
+                    button.innerHTML = 'load brent oil history';
+                    button.disabled = false;
+                    return;
+                }
+                var stringJson;
+                var numberOfValues = 0;
+
+                var jsonValues = JSON.parse(values, function (key, value) {
+                    if(key == "id"){
+                        numberOfValues++;
+                        return new Date(Date.parse(value.price_timestamp));
+                    } else{
+                        return value;
+                    }
+                });
+
+                stringJson = JSON.stringify(jsonValues);
+                console.log(stringJson);
+
+                var data = new google.visualization.DataTable();
+                // Add columns
+                data.addColumn('date', 'Date');
+                data.addColumn('number', 'rate');
+                for(var i=0;i<numberOfValues;i++){
+                    data.addRow([jsonValues[i].id,jsonValues[i].price]);
+                }
+
+
+                var options = {
+                    title: 'Brent Oil Currencies',
+                    curveType: 'function',
+                    legend: {position: 'bottom'},
+                    hAxis: {
+                        title: "dates"
+                    },
+                    vAxis: {
+                        title: "Values"
+                    }
+                };
+
+                var chart = new google.visualization.AreaChart(document.getElementById('curve_chart'));
+
+                chart.draw(data, options);
```

Comment:
As discussed - would be cool to have both predicted and historical prices on a single chart.
I really think it would be a killer feature